nac_collector/github_repo_wrapper.py

```diff
@@ -95,8 +95,7 @@ def get_definitions(self):
                 files, label="Processing terraform provider definitions"
             ) as files_bar:
                 for file in files_bar:
-                    # Exclude *_update_rank used in ISE from inspecting
```

Comment:
this if statement is for ISE resources which should be excluded, and this line should stay there, otherwise its gonna fail for ISE